File: Logical_operator/2.py
ID_user = input("사용자 계정을 입력하세요.\n")
PWD_user = input("패스워드를 입력하세요.\n")

ID_Sinu = "sinu"
PWD_Sinu = "1234567"

ID_Kimgeek = "Kimgeek"
PWD_Kimgeek = "4242424242"

if ID_user == ID_Sinu and PWD_user == PWD_Sinu :
    print("다시 오신 걸 환영합니다! Sinu님.")
elif ID_user == ID_Kimgeek and PWD_user == PWD_Kimgeek :
    print("다시 오신 걸 환영합니다! Kimgeek님.")
else :
    print("이런! 사용자 계정 또는 패스워드가 올바르지 않습니다!")
    pass

File: Logical_operator/2.rb
puts("아이디를 입력하세요.")
ID_user = gets.chomp()
puts("패스워드를 입력하세요.")
PWD_user = gets.chomp()

ID_Sinu = "Sinu"
PWD_Sinu = "I came, I saw, I conquered."

ID_Kimgeek = "Kimgeek"
PWD_Kimgeek = "Remarkable"

if ID_user == User_Sinu and PWD_user == PWD_Sinu
  puts("다시 오셔서 기뻐요! "+ID_user+"님.")
elsif ID_user == User_Kimgeek and PWD_user == PWD_Kimgeek
  puts("다시 오셔서 기뻐요! "+ID_user+"님.")
else
  puts("아이디 또는 패스워드가 정확하지 않습니다!")
end
